cmd/tracee-ebpf/internal/flags/flags-filter.go

```diff
@@ -143,10 +144,11 @@ func PrepareFilter(filters []string) (tracee.Filter, error) {
 	eventFilter := &tracee.StringFilter{Equal: []string{}, NotEqual: []string{}}
 	setFilter := &tracee.StringFilter{Equal: []string{}, NotEqual: []string{}}
 
-	eventsNameToID := make(map[string]int32, len(tracee.EventsDefinitions))
-	for id, event := range tracee.EventsDefinitions {
-		if !event.Internal {
-			eventsNameToID[event.Name] = id
+	eventsNameToID := events.Definitions.NamesToIDs()
+	// remove internal events (not sure why if someone knows please fix the comment)
```

Comment:
From what I see currently no error returns, we just have a different form of ignore logic (I explicitely remove from a copy, instead of adding to the copy only what's not internal)